ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/accordion/v1/accordion/_cq_dialog/.content.xml

```diff
@@ -47,18 +47,31 @@
                         jcr:title="Accessibility"
                         sling:resourceType="granite/ui/components/coral/foundation/container">
                         <items jcr:primaryType="nt:unstructured">
-                            <assistPriority
-                                jcr:primaryType="nt:unstructured"
-                                sling:resourceType="granite/ui/components/coral/foundation/include"
-                                path="core/fd/components/form/base/v1/base/cq:dialog/content/items/tabs/items/accessibility/items/assistPriority"/>
-                            <customText
-                                jcr:primaryType="nt:unstructured"
-                                sling:resourceType="granite/ui/components/coral/foundation/include"
-                                path="core/fd/components/form/base/v1/base/cq:dialog/content/items/tabs/items/accessibility/items/customText"/>
-                            <roleAttribute
-                                jcr:primaryType="nt:unstructured"
-                                sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
-                                granite:hide="{Boolean}true"/>
+                            <columns
+                                    jcr:primaryType="nt:unstructured"
+                                    sling:resourceType="granite/ui/components/coral/foundation/fixedcolumns"
+                                    margin="{Boolean}true">
+                                <items jcr:primaryType="nt:unstructured">
+                                    <column
+                                            jcr:primaryType="nt:unstructured"
+                                            sling:resourceType="granite/ui/components/coral/foundation/container">
+                                        <items jcr:primaryType="nt:unstructured">
+                                            <assistPriority
+                                                    jcr:primaryType="nt:unstructured"
+                                                    sling:resourceType="granite/ui/components/coral/foundation/include"
+                                                    path="core/fd/components/form/base/v1/base/cq:dialog/content/items/tabs/items/accessibility/items/columns/items/column/items/assistPriority"/>
```

Comment:
minor: pls fix the indentation.

Reply:
The indentation is correct, due to long text it is showing in next line here.